test/Microsoft.Azure.Devices.Edge.Test.Common/Process.cs

```diff
@@ -1,15 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace Microsoft.Azure.Devices.Edge.Test.Common
 {
+    using System;
+    using System.Collections.Generic;
     using System.ComponentModel;
     using System.Diagnostics;
     using System.Threading;
     using System.Threading.Tasks;
     using RunProcessAsTask;
+    using Serilog;
```

Comment:
Do we need `Serilog` here?

Reply:
Yes, because I added two calls to `Log.Verbose(...)`.